dev/tools/gen_keycodes/lib/ios_code_gen.dart

```diff
@@ -75,17 +108,37 @@ class IOSCodeGenerator extends PlatformCodeGenerator {
   String get _keyToModifierFlagMap {
     final StringBuffer modifierKeyMap = StringBuffer();
     for (final String name in kModifiersOfInterest) {
-      final String line = '{${toHex(logicalData.entryByName(name).iOSKeyCodeValues[0])}, kModifierFlag${lowerCamelToUpperCamel(name)}},';
+      final String line ='    {${toHex(logicalData.entryByName(name).iOSKeyCodeValues[0])}, kModifierFlag${lowerCamelToUpperCamel(name)}},';
       modifierKeyMap.writeln('    ${line.padRight(42)}// $name');
     }
     return modifierKeyMap.toString().trimRight();
   }
 
+  /// This generates a map from a sided modifier (e.g. LeftShift) to the
+  /// equivalent unsided modifier (e.g. ShiftAny).
+  String get _sidedModifierToAnyMap {
+    final StringBuffer modifierKeyMap = StringBuffer();
+    for (final String name in sidedModifierToAny.keys) {
+      modifierKeyMap.writeln('    {kModifierFlag$name, kModifierFlag${sidedModifierToAny[name]!}},');
+    }
+    return modifierKeyMap.toString().trimRight();
+  }
+
+  /// This generates a map from an unsided modifier (e.g. ShiftAny) to the
```

Comment:
When is `ShiftAny` used? I see you added them to maps, but are they used in anywhere output? Or are they just intermediate values?

Reply:
Yeah, I did have a use for those at one point, but I seem to have since factored it out. Removed the unneeded maps.